### PR TITLE
New Landing Page

### PR DESCRIPTION
**Pull request by harshit1142**
Added a Responsive Movie Landing Page

Please add hacktoberfest label

### CODE REVIEW

File: MovieLanding Page/index.html [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+        <div class="menu">
+             <div class="line"></div>
+             <div class="line"></div>
+             <div class="line"></div>
+            
+        </div>
+        <div class="nav-section">
+            <ul>
+                <li >
+                    <a href="#highlight " id="navi" class="link">Home</a>
```
Comment (0silverback0) [inline MovieLanding Page/index.html]: Should link to /index.html also

**Review by 0silverback0 — CHANGES_REQUESTED** [review]

File: MovieLanding Page/index.html (hunk 1 above)
Comment (harshit1142) [reply MovieLanding Page/index.html]: Okay wait.

**Comment by harshit1142** [comment]
Done

**Comment by 0silverback0** [comment]
please pull the lates changes and commit

**Comment by harshit1142** [comment]
Please Add Hacktoberfest label if possilble.

**Pull request closed without merging**